core/src/km_core_action_api.cpp

```diff
@@ -30,71 +28,32 @@ km_core_actions const * km_core_state_get_actions(
     return nullptr;
   }
 
-  auto action_items = km_core_state_action_items(state, nullptr);
```

Comment:
This code has moved essentially without change into prepare_actions() in state.cpp